internal/page/app_data.go

```diff
@@ -26,6 +26,7 @@ type AppData struct {
 	ActorType        actor.Type
 	AttorneyID       string
 	OneloginURL      string
+	AppPublicURL     string
```

Comment:
This felt like a natural place for this to live and have handy. I've not extended to packages other than `donor` yet but can do so in a follow-up PR if we're happy with it.